components/brave_wallet/browser/rlp_encode_unittest.cc

```diff
@@ -21,7 +21,6 @@ base::Value RLPTestStringToValue(const std::string& s, std::string* remaining) {
   State state = Start;
   while (*p != '\0') {
     if (isdigit(*p) && state == Start) {
-      state = InNumber;
```

Comment:
why are we removing this?

Reply:
This branch will always return without reading this value.

Reply:
I see. Shouldn't we also remove it from enum?

Reply:
Actually yes... well caught. Fixed.